chore: add changelog to 12.17.3 (#32680)
MetaMask/metamask-extension: CHANGELOG.md

## [Unreleased]

## [12.17.3]

> ### Fixed

Reviewer: There was one more change not listed:

```suggestion
### Changed
- Update supported browser versions ([#32520](https://github.com/MetaMask/metamask-extension/pull/32520))
  - The minimum supported browser versions are now Chromium v109 and Firefox v102
  - The "outdated browser" in-app prompt now asks users below Chromium v113 or Firefox v115 to update
 
### Fixed
```

- Stop emitting Dapp Viewed events on Firefox ([#32583](https://github.com/MetaMask/metamask-extension/pull/32583))
- Make phishing warning events anonymous ([#32635](https://github.com/MetaMask/metamask-extension/pull/32635))

> - Keep showing the approve row in the batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))

Reviewer: Nit: "keep showing" makes it unclear exactly what was fixed. I've attempted to re-word it to make it more clear that it's fixing a bug where the approve row disappears briefly.
```suggestion
- Fix momentary disappearance of approve row in batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))
```

Author: Makes sense, thanks for the suggestion !

- Support permit2 approvals in batch simulation ([#32733](https://github.com/MetaMask/metamask-extension/pull/32733))

> - Add metrics to check rejection of smart account update by user ([#32737](https://github.com/MetaMask/metamask-extension/pull/32737))

Reviewer: Nit: This one is us fixing a metric event that we use internally. Not something with a user impact, should be safe to omit it here.

```suggestion
```

> - Delete preference to persist user rejection of upgrade ([#32697](https://github.com/MetaMask/metamask-extension/pull/32697))

Reviewer: Not entirely happy with my wording choice here, but here's an attempt to explain that we're allowing users to upgrade who rejected the upgrade in the past:
```suggestion
- Allow users to upgrade to a smart contract account after rejecting it once in the past ([#32697](https://github.com/MetaMask/metamask-extension/pull/32697))
```

The previous wording was a little confusing to me (it says we're "deleting a preference", but the preference itself isn't something user-facing, it's strictly internal)
